packages/reaction-product-variant/client/templates/products/productDetail/productDetail.js

```diff
@@ -199,17 +199,20 @@ Template.productDetail.events({
       template.$(".title-edit-input").focus();
     }
     const variants = ReactionProduct.getVariants(self._id);
-    for (let variant of variants) {
-      let index = _.indexOf(variants, variant);
+    variants.forEach((variant, index) => {
```

Comment:
[for of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) works for for any iterable object such as an Array, String, TypedArray, Map, Set, generators, and NodeList; but necessarily every kind of Object. 

The usage `Array.forEach` can work too, especially, if you need to access the index.

In general. Prefer `for of`.